### PR TITLE
Update account card

### PR DESCRIPTION
**Pull request by serjonya-trili**
## Proposed changes

[Task link](https://app.asana.com/0/1207041888417083/1207877246316768/f)

## Types of changes

- [ ] Bugfix
- [ ] New feature
- [x] Refactor
- [ ] Breaking change
- [x] UI fix

## Screenshots

Add the screenshots of how the app used to look like and how it looks now

| Before | Now |
| ------ | --- |
|   Screenshot 2024-07-31 at 09 28 42 | Screenshot 2024-07-31 at 09 28 35  |

## Checklist

- [x] Tests that prove my fix is effective or that my feature works have been added
- [ ] Documentation has been added (if appropriate)
- [x] Screenshots are added (if any UI changes have been made)
- [ ] All TODOs have a corresponding task created (and the link is attached to it)

### CODE REVIEW

File: apps/web/src/components/AccountCard/AccountBalance.tsx [hunk 1]
```diff
@@ -26,37 +30,29 @@ export const AccountBalance = () => {
         >
           Tez Balance
         </Text>
-        <Text color={color("900")} fontWeight="600" size="2xl">
-          {balance && prettyTezAmount(balance)}
-        </Text>
-        <Text color={color("700")} size="sm">
-          {usdBalance && `${usdBalance}$`}
-        </Text>
+        {balance !== undefined && (
```
Comment (OKendigelyan) [inline apps/web/src/components/AccountCard/AccountBalance.tsx:33]: Will it work like `balance && ...` and below too?

Comment (OKendigelyan) [inline apps/web/src/components/AccountCard/AccountBalance.tsx:33]: Will it work like `balance && ...` and below too?

File: apps/web/src/components/AccountSelectorModal/AccountSelectorModal.tsx [hunk 2]
```diff
@@ -60,7 +60,7 @@ export const AccountSelectorModal = () => {
                   paddingLeft="12px"
                 >
                   <Heading color={color("900")} size="sm">
-                    {capitalize(type)}
+                    {type.split("_").map(capitalize).join(" ")}
```
Comment (OKendigelyan) [inline apps/web/src/components/AccountSelectorModal/AccountSelectorModal.tsx:63]: Maybe it would be better to move it to a separate variable/function?

**Review by OKendigelyan — APPROVED** [review]
Approve, just have some minor comments

File: apps/web/src/components/AccountCard/AccountBalance.tsx (hunk 1 above)
Comment (serjonya-trili) [reply apps/web/src/components/AccountCard/AccountBalance.tsx:33]: not exactly. if the balance is 0 then it'll show just 0, not 0 Tez

File: apps/web/src/components/AccountSelectorModal/AccountSelectorModal.tsx (hunk 2 above)
Comment (serjonya-trili) [reply apps/web/src/components/AccountSelectorModal/AccountSelectorModal.tsx:63]: created a [ticket](https://app.asana.com/0/1207041888417083/1207877246316773/f) for that

**Pull request merged**